x-pack/legacy/plugins/maps/public/actions/map_actions.js

```diff
@@ -645,15 +646,25 @@ function removeLayerFromLayerList(layerId) {
   };
 }
 
-export function setQuery({ query, timeFilters, filters = [] }) {
+export function setQuery({ query, timeFilters, filters = [], refresh = false }) {
```

Comment:
not sure if we want to hide this bug-fix as a default-param (shouldn't the default actually be `true` anyway, since it's the default behavior?).

I think it would improve readability if we add

```
refresh: false
```

here https://github.com/elastic/kibana/blob/77a9e6aaeb4fa9b2bc5f2242d20efedc805992f3/x-pack/legacy/plugins/maps/public/angular/map_controller.js#L235 

and introduce a `refresh = false` further up here:


https://github.com/elastic/kibana/blob/77a9e6aaeb4fa9b2bc5f2242d20efedc805992f3/x-pack/legacy/plugins/maps/public/embeddable/map_embeddable.js#L75

Reply:
I could go either way with the refresh parameter. I just made the default false since refresh is the special state since embeddable has a reload method and triggers it and search_bar has onSubmit that triggers refresh. 

How strongly do you feel about this change?

Reply:
I don't feel strongly. Is good as is, just took me a while to wrap my head around the flow.